ui/admin/app/templates/scopes/scope/workers/index.hbs

```diff
@@ -145,6 +149,56 @@
           </B.Tr>
         </:body>
       </Hds::Table>
+
+      {{#if this.selectedWorker}}
+        <Hds::Flyout
+          data-test-worker-tags-flyout
+          @onClose={{fn this.selectWorker null}}
+          as |M|
+        >
+          <M.Header>
+            {{t
+              'resources.worker.flyout.title'
+              workerName=this.selectedWorker.displayName
+            }}
+          </M.Header>
+          <M.Body>
+            <Hds::Table
+              @model={{this.selectedWorkerTags}}
+              @columns={{array
+                (hash label=(t 'resources.worker.flyout.table.tag'))
+                (hash label=(t 'resources.worker.flyout.table.type'))
+              }}
+              @valign='middle'
+            >
+              <:body as |B|>
+                <B.Tr>
+                  <B.Td>
+                    <Hds::Text::Code>
+                      {{this.tagDisplayName B.data}}
+                    </Hds::Text::Code>
+                  </B.Td>
+                  <B.Td>
+                    <Hds::Badge @text={{B.data.type}} />
+                  </B.Td>
+                </B.Tr>
+              </:body>
+            </Hds::Table>
+            {{#if (eq this.selectedWorkerTags.length 10)}}
```

Comment:
I think this should be slightly updated. The "View more tags" link should be displayed if there are more than 10 tags. So in the case that there are only 10 tags, the link will still be displayed.

Reply:
Great point. Updated and will push up shortly.